docs/lib/DEBUGGING.md

```diff
@@ -165,17 +155,16 @@ static void halt_error(int reason) {
     NVIC_SystemReset();
 }
 
-static void box_debug_main(const void *)
-{
-    /* Debug box driver -- Version 0 */
-    static const TUvisorDebugDriver driver = {
-        get_version,
-        halt_error
-    };
+/* Create a valid debug driver struct and use example_halt_error() as its halt_error() function */
```

Comment:
It's in the scope of Oren's work, yes. I don't believe it has been done by any PR from Oren yet, though.